### ISSUE TITLE
Add Support for Accessibility Props in <Trans> Component

### ISSUE THREAD

**Issue by ud7811** [i18next/react-i18next#1853]
## 🚀 Feature Proposal

In React Native, it currently lacks support for passing accessibility props such as accessibilityLabel, accessibilityRole and importantForAccessibility. This limits its use in building accessible mobile application.

## Motivation

Accessibility is essential in mobile application development, especially for users on screen readers like VoiceOver (iOS) or talkback (Android). React native provides a rich set of accessibility props, but the current <Trans> implementation does not support passing these props directly to the rendered elements.

Adding support for accessibility props would:
1. Improve inclusivity and usability of internationalized components.
2. Reduce the need for workarouns or wrapper components.
3. Align with React Native's accessibility standards.

## Example

<Trans
 i18nKey="welcomeMessage"
 accessibilityLabel="Welcome message for screen readers"
 accessibilityRole="button"
>
Welcome to our app!
</Trans>

**Comment by adrai** [comment]
Hi @ud7811 

accessibility is an important and interessant topic. The question related to the Trans component is: What should happen with those properties?
By default the <Trans> component does not wrap anything in the resulting ui... it just returns the Trans children or the corresponding i18n key value etc...
There is a `transWrapTextNodes` option, but I don't think this is the way to go.

**Comment by ud7811** [comment]
Hi @adrai 
Thanks for the input! One idea could be to allow an optional wrapperComponent (like a Text or View) so accessibility props can be passed to it. Would this be a direction worth exploring?

**Comment by adrai** [comment]
Like said before, you can do this directly in userland...
In your Trans component, add whatever inner component you like.

Wrap trans:

```jsx
<WrapperView>
  <Trans></Trans>
</WrapperView>
```

or wrap in trans:

```jsx
<Trans>
  <WrapperView>
    <Text></Text>
  </WrapperView>
</Trans>
```
